Parser/asdl_c.py

```diff
@@ -1150,8 +1150,194 @@ def visitModule(self, mod):
     {NULL}
 };
 
+static PyObject *
+ast_repr_max_depth(AST_object *self, int depth);
+
+static PyObject *
+ast_repr_list(PyObject *list, int depth)
```

Comment:
Maybe add a comment saying that it will only show the first/last item of the list? If you want a more generic function, you can even specify an (event) width of the list to show (here, your function would be with width = 2).

Reply:
Added a comment to the function :) I don't think making the function generic is particularly useful since there's no way the user can modify the value. Though if it's needed at some point, it should be fairly easy to add